src/containers/Network/BarChartVersion.tsx

```diff
@@ -129,6 +137,7 @@ const BarChartVersion = (props: Props) => {
               backgroundColor: GREY_600,
               borderRadius: 8,
               border: ['1px solid', GREY_800].join(' '),
+              opacity: showTooltips ? '100%' : '0',
```

Comment:
This might end up being confusing because there are multiple ways of indicating whether a tooltip should appear (Even if you do the function, the opacity will basically override it) - Maybe it'd be better to have a single method of deciding to show/hide the tooltip and using it across the board?

Reply:
@ckniffen

Reply:
What are the other places?  Also this spot uses recharts Tooltip.